mp/game/momentum/custom/hud/materials/sprites/crosshair_custom.vmt

```diff
@@ -0,0 +1,10 @@
+"UnlitGeneric"
+{
+    "$basetexture" "sprites/crosshair_custom"
+    "$translucent"   "1"
+    "$translucency" "1"
+    "$ignorez" "1"
+    "$nomip" "1"
+    "$nocompress" "1"
+    "$nolod" "1"
+}
```

Comment:
If something is going to ship with the game, it should be in the game's folder. The custom folder is meant to be only for the end user, meaning if they want to override this custom crosshair, they would use that folder as it gets mounted above our game's folder. Currently with this, if they override this file, every time they validate files it would be overwritten with this, which is not good.

Can you move this to the game's materials folder?